Kudu.Core/SiteExtensions/FeedExtensions.cs

```diff
@@ -260,6 +266,9 @@ private static void WriteStreamToFile(Stream stream, string filePath)
             FileSystemHelpers.EnsureDirectory(packageFolderPath);
             using (Stream writeStream = FileSystemHelpers.OpenWrite(filePath))
             {
+                // reset length of file stream
+                writeStream.SetLength(0);
```

Comment:
i think line 270 should be moved to right before "stream.CopyTo" where retry actually happen